tvpeter/currency-converter
# ch(setup project): configure webpack and an empty react project

#### What does this PR do?

- Setup our currency converter React App

#### What are the tasks to be completed

- Creates a react app from scratch
- configure `formatting, linting and compiling` using prettier, eslint and webpack

#### How can this be tested manually

- Clone this repo and run `npm i` to install the dependencies
- run `npm start` to start the server and the app will launch and you will see it in the browser.

#### Any related pivotal stories?

- [#168530477](https://www.pivotaltracker.com/story/show/168530477)

#### Screenshots?

-N/A
